src/pages/settings/Profile/ProfilePage.js

```diff
@@ -213,9 +213,9 @@ class ProfilePage extends Component {
                         </Text>
                         <FullNameInputRow
                             firstName={this.state.firstName}
-                            firstNameError={PersonalDetails.getMaxCharacterError(this.state.firstNameError)}
+                            hasFirstNameError={PersonalDetails.getMaxCharacterError(this.state.firstNameError)}
                             lastName={this.state.lastName}
-                            lastNameError={PersonalDetails.getMaxCharacterError(this.state.lastNameError)}
+                            hasLastNameError={PersonalDetails.getMaxCharacterError(this.state.lastNameError)}
```

Comment:
This was not what I meant. 

`FullNameInputRow` will accept error strings only, In the future, there could be different error strings, so we want to keep it dynamic. 

Instead
` firstNameError={PersonalDetails.getMaxCharacterError(this.state.firstNameError)}`
 =>
 `firstNameError={PersonalDetails.getMaxCharacterError(this.state.hasFirstNameError)}`